src/components/SignatureLoadingModal.tsx

```diff
@@ -17,7 +17,6 @@ const SignatureLoading = ({
   return (
     <>
       <div className='text-2xl font-semibold'>Transaction Pending...</div>
-      {/*  TODO: add a spinner */}
```

Comment:
I actually just wanted a spinner not a new modal sorry for the confusion.

Reply:
Oh I see, just add a spinner for Signature Loading modal